### PR TITLE
CVE command

### PR DESCRIPTION
**Pull request by coreybodendein-cyware**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add new `ctix-get-vulnerability-data` and `cve` commands to CTIXv3 integration.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CTIX/Integrations/CTIXv3/CTIXv3.py [hunk 1]
```diff
@@ -1924,6 +1968,102 @@ def make_request(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
...
+            )
+            source_description = response.get("data", {}).get("result", {})
+            if source_description:
+                description = source_description.get("description")
+                if description:
+                    break
+
+    cve_standard_context = Common.CVE(name, cvss2, created, modified, description)
+    data = {
+        "cpes": cpe_list if cpe_list else "None",
```
Comment (thefrieddan1) [inline Packs/CTIX/Integrations/CTIXv3/CTIXv3.py:2046]: Can use a simpler:
 "cpes": cpe_list or "None"

**Comment by thefrieddan1** [comment]
Hi @coreybodendein-cyware thank you for your contribution we appreciate your effort, I reviewed the content in the PR and would like to schedule a demo.

**Comment by thefrieddan1** [comment]
Hi @coreybodendein-cyware ,


Please see the following error in the validations: 

`No release note entry was found for the integration "Cyware Threat Intelligence eXchange" in the CTIX pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items`

File: Packs/CTIX/Integrations/CTIXv3/CTIXv3.py [hunk 2]
```diff
@@ -1924,6 +1968,102 @@ def make_request(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
             ]
 
 
+def cve_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    page = 1
+    page_size = 15
+    params = {"page": page, "page_size": page_size}
+    cve = args["cve"]
+    if isinstance(cve, str):
```
Comment (thefrieddan1) [inline Packs/CTIX/Integrations/CTIXv3/CTIXv3.py:1976]: Please use argToList on cve instead of split

Comment (coreybodendein-cyware) [reply Packs/CTIX/Integrations/CTIXv3/CTIXv3.py]: Thanks! I made the change as you suggested.

**Comment by coreybodendein-cyware** [comment]
@thefrieddan1, I made some further changes to existing commands in c3048032fc7ee9fb6c346de9580186c366d938d1 that have parameters that are actually required, i.e. they use `args["arg_name"]`, which resulted in the below error while validating the yaml file:

```
Validating backwards compatibility for Packs/CTIX/Integrations/CTIXv3/CTIXv3.yml
[ERROR][INFO] : Packs/CTIX/Integrations/CTIXv3/CTIXv3.yml: [BC104] - Possible backwards compatibility break, Your updates to this file contains changes to a name or an argument of an existing command(s).
Please undo you changes to the following command(s):
ctix-get-saved-searches
ctix-get-server-collections
ctix-get-actions
ctix-add-tag-indicator
ctix-remove-tag-from-indicator
ctix-get-indicator-details
ctix-get-indicator-tags
ctix-get-indicator-relations
```

Note that I didn't change any behavior in the actual python code, this change just reflects the reality that the commands would error out if those required parameters weren't actually provided.

What do you recommend for this? I suppose I could remove the `.yml` changes which would get past the CI check and leave the corresponding `README` changes to reflect that these parameters are required.

**Comment by coreybodendein-cyware** [comment]
> Hi @coreybodendein-cyware thank you for your contribution we appreciate your effort, I reviewed the content in the PR and would like to schedule a demo.

Hi @thefrieddan1, let me know how you would like to me to schedule the demo.

**Comment by thefrieddan1** [comment]
Hi @coreybodendein-cyware sorry about the delay,
We are still looking for the best way to proceed with the BC104 error.

Thanks

**Comment by thefrieddan1** [comment]
Hi @coreybodendein-cyware 
After much discussions :) 
We have decided on a preferred method of action please remove the added `required` and add a `defaultValue` field in the yml.

Thanks

**Comment by coreybodendein-cyware** [comment]
> Thanks @coreybodendein-cyware for the quick implementation. Can you please give me permissions to this branch ? I would like to bump the docker versions and update the ReleaseNote. Thanks

Hi @thefrieddan1, I took care of that.

File: Packs/CTIX/Integrations/CTIXv3/README.md [hunk 3]
```diff
@@ -119,853 +127,888 @@ Get paginated list of tags
...
-| page_size | total number of results to be fetched. Default is 10. | Optional | 
-| q | query. | Optional | 
-| object_id | object id. | Optional | 
-| object_type | object type. | Optional | 
-| tag_id | tag id. | Optional | 
-
+| page | page from where data will be taken. Default is 1. | Optional |
+| page_size | total number of results to be fetched. Default is 10. | Optional |
+| q | query. | Optional |
+| object_id | object id. | Required |
```
Comment (thefrieddan1) [inline Packs/CTIX/Integrations/CTIXv3/README.md:842]: please update README with the current changes. 
no changes were made to Optional arguments, they are marked as Optional but with defaultValue

File: Packs/CTIX/ReleaseNotes/2_2_10.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#### Integrations
+
+##### Cyware Threat Intelligence eXchange
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+
+##### CTIX v3
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+- Added a ctix-get-vulnerability-data command.
+- Added a cve command.
+- Update documentation for fields that are actually required.
```
Comment (thefrieddan1) [inline Packs/CTIX/ReleaseNotes/2_2_10.md:10]: Perhaps "Added a default value for required argument fields" ?

**Review by thefrieddan1 — COMMENTED** [review]
Lets please schedule a demo.
Im on GTM+3.
We need half an hour.
Available today from 13:00 to 14:00. 
Or on Monday 11:30 to 14:00 or from 15:00 to 16:00.
If you can schedule for later hours that will be possible on Tuesday 

Thanks

File: Packs/CTIX/ReleaseNotes/2_2_10.md (hunk 4 above)
Comment (coreybodendein-cyware) [reply Packs/CTIX/ReleaseNotes/2_2_10.md:10]: Thanks, I made that change.

File: Packs/CTIX/Integrations/CTIXv3/README.md (hunk 3 above)
Comment (coreybodendein-cyware) [reply Packs/CTIX/Integrations/CTIXv3/README.md:842]: I went ahead and updated the README to reflect the optional arguments with default values.

**Comment by coreybodendein-cyware** [comment]
> Lets please schedule a demo. Im on GTM+3. We need half an hour. Available today from 13:00 to 14:00. Or on Monday 11:30 to 14:00 or from 15:00 to 16:00. If you can schedule for later hours that will be possible on Tuesday
> 
> Thanks

I'm GMT-5 so we're quite a ways off on timezones. Later hours for you on Tuesday would be my morning so that would be ideal. If you could email me at corey.bodendein@cyware.com we can get an invite on both of our calendars, thanks!

**Review by thefrieddan1 — APPROVED** [review]

**Pull request merged**